idd/Energy+.idd.in

```diff
@@ -91303,809 +91272,129 @@ Curve:ChillerPartLoadWithLift,
-  N7 , \field Output Value #1
-       \required-field
-       \type real
-  N8 , \field X Value #2
+
+Table:IndependentVariableList,
+       \memo A sorted list of independent variables used by one or more Table:Lookup
+       \memo objects.
+       \extensible:1
+  A1,  \field Name
        \required-field
-       \type real
-  N9 , \field Output Value #2
+       \reference IndependentVariableListName
+       \type alpha
+  A2;  \field Independent Variable 1 Name
        \required-field
-       \type real
-  N10, \field X Value #3
-       \type real
-  N11, \field Output Value #3
-       \type real
-  N12, \field X Value #4
-       \type real
-  N13, \field Output Value #4
-       \type real
-  N14, \field X Value #5
-       \type real
-  N15, \field Output Value #5
-       \type real
-  N16, \field X Value #6
-       \type real
-  N17, \field Output Value #6
-       \type real
-  N18, \field X Value #7
-       \type real
-  N19, \field Output Value #7
-       \type real
-  N20,N21,N22,N23,N24,N25,N26,N27,N28,N29,N30,N31,N32,N33,N34,N35,N36,N37,N38,N39, \note fields as indicated
-  N40,N41,N42,N43,N44,N45,N46,N47,N48,N49,N50,N51,N52,N53,N54,N55,N56,N57,N58,N59, \note fields as indicated
-  N60,N61,N62,N63,N64,N65,N66,N67,N68,N69,N70,N71,N72,N73,N74,N75,N76,N77,N78,N79, \note fields as indicated
-  N80,N81,N82,N83,N84,N85,N86,N87,N88,N89,N90,N91,N92,N93,N94,N95,N96,N97,N98,N99, \note fields as indicated
-  N100,N101,N102,N103,N104,N105,N106,N107,N108,N109,N110,N111,N112,N113,N114,N115,N116,N117,N118,N119; \note fields as indicated
+       \type object-list
+       \object-list IndependentVariableName
+       \begin-extensible
 
-Table:TwoIndependentVariables,
-       \extensible:3 repeat last three fields
-       \min-fields 14
-       \memo Allows entry of tabular data pairs as alternate input
-       \memo for performance curve objects.
-       \memo Performance curve objects can be created using these inputs.
-       \memo BiQuadratic Table Equation: Output = a + bX + cX**2 + dY + eY**2 + fXY
-       \memo BiQuadratic solution requires a minimum of 6 data pairs
-       \memo QuadraticLinear Table Equation: Output = a + bX + cX**2 + dY + eXY + fX**2Y
-       \memo QuadraticLinear solution requires a minimum of 6 data pairs
-  A1 , \field Name
+Table:Lookup,
+       \memo Lookup tables are used in place of curves and can represent any number
+       \memo of independent variables (defined as Table:IndependentVariable objects
+       \memo in a Table:IndependentVariableList). Output values are interpolated
+       \memo within the bounds defined by each independent variable and extrapolated
+       \memo beyond the bounds according to the interpolation/extrapolation methods
+       \memo defined by each independent variable.
+       \extensible:1
+  A1,  \field Name
        \required-field
+       \reference AllCurves
+       \reference UniVariateTables
+       \reference BiVariateTables
+       \reference MultiVariateTables
```

Comment:
These should be UniVariateFunctions, BiVariateFunctions, MultiVariateFunctions to align with the rest of the IDD.

Reply:
Good catch. Also need to add tri- and quad- variatefunctions here.